src/main/php/StreamShortHandler.php

```diff
@@ -3,40 +3,23 @@
 
 namespace Damejidlo\CrashLog;
 
+use Monolog\Handler\StreamHandler;
 use Tracy\Dumper;
 use Tracy\Helpers;
-use Tracy\ILogger;
 
 
 
-class StreamShortLogger implements ILogger
+class StreamShortHandler extends StreamHandler
 {
 
-	/**
-	 * @var ILogger
-	 */
-	private $delegate;
-
-	/**
-	 * @var string
-	 */
-	private $destination;
-
-
-
-	public function __construct(ILogger $delegate, string $destination)
+	protected function streamWrite($stream, array $record)
 	{
-		$this->delegate = $delegate;
-		$this->destination = $destination;
-	}
-
+		// log only exceptions
+		if (isset($record['context']['exception'])) {
```

Comment:
Toto by se mělo řídit prioritou logované zprávy, ne přítomností `exception` parametru v kontextu (ten bude stejně odstraněn pomocí procesoru výše)... a to zvládá dostatečně obstojně existující `StreamHandler` z Monologu ;)